_episodes/09-github.md

```diff
@@ -257,12 +257,20 @@ f22b25e Start notes on Mars as a base
 ~~~
 {: .output}
 
-Finally, there is an very handy Git command called `clone`. In one step this command
-can bring us to the current state of our freshly made repository. Let's create a
-new directory outside of our current repository and try this command. 
+Finally, there is an very handy Git command called `clone`
+
+This command will perform all of the steps above in one line: 
+
+  1. make a new directory with the repository's name
+  2. initialize a bare repository
```

Comment:
Sorry to be a stickler but I think this should be empty again.  But all the other changes look good